journals/apps/journals/templates/wagtailadmin/home.html

```diff
@@ -0,0 +1,4 @@
+{% extends "wagtailadmin/home.html" %}
+{% load i18n %}
+
+{% block branding_welcome %}{% blocktrans %}Welcome to the {{ site_name }} Journal Editor{% endblocktrans %}{% endblock %}
```

Comment:
Change this to `Welcome to the edX Journal Editor`
remove the {{ site_name }} as that evaluates to "Journals" and is read from base.py in settings.